packages/server/src/sdk/app/rows/search/external.ts

```diff
@@ -133,32 +133,32 @@ export async function exportRows(
 
   let result = await search({ tableId, query: requestQuery, sort, sortOrder })
   let rows: Row[] = []
+  let headers
 
-  // Filter data to only specified columns if required
+  if (!tableName) {
+    throw new HTTPError("Could not find table name.", 400)
```

Comment:
nit: please use `BadRequestError` instead of `HTTPError` here. It represents the intent slightly better.